load-execute.sh

```diff
@@ -4,8 +4,8 @@ set -e
 start_api() {
     echo 'Starting API'
     export DB_URL=localhost
-    export DB_PORT=5433
-    export DB_NAME=intermediary
+    export DB_PORT=5434
```

Comment:
Why was this port changed to 5434?

Reply:
so that if you have the regular local DB already up and running, this won't conflict with or kill it